### PR TITLE
Ask representation

### LINKED ISSUE #35: Implement display for ASK queries

### PR DESCRIPTION
**Pull request by AronBuzogany**
This PR fixes #35 : 

- Made display for ASK query results
- Users can configure the text that gets shown when the result is true and when the result is false. This is more user friendly than simply true or false. 
- An example query showcasing the results of this PR

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,7 +38,9 @@ The configuration file follows a simple structure.
             "name": "A name for the query",
             "description": "Description of the query",
             "id": "A unique ID for the query",
-            "sources": "Sources over which the query should be executed"
+            "sources": "Sources over which the query should be executed",
+            "trueText": "The text that is to be shown when the query result is true, only useful for ASK queries.",
```
Comment (pheyvaer) [inline README.md:42]: In this case I would add an object like:
```json
"askQuery": {
  "trueText": "...",
  "falseText": ",,,"
}
```

File: public/queries/ask_picasso.ttl
Comment (pheyvaer) [inline public/queries/ask_picasso.ttl:1]: The extension is `ttl` but that is reserved for Turtle files. This is not a Turtle file. The extension for files with SPARQL queries is `rq`.

File: src/right-field/RightField.jsx [hunk 2]
```diff
@@ -240,24 +253,35 @@ async function executeQuery(query, eventEmitter) {
  * A function that given a QueryType processes every result as a stream.
  *
  * @param {QueryType} execution a query execution
+ * @param {query} query the query which is being executed
  * @param {EventEmitter} eventEmitter an EventEmitter that listens to and emits UI state changes.
+ * @param {Function} adder function that sets UI state of the query result
  */
-async function handleQueryExecution(execution, eventEmitter) {
+async function handleQueryExecution(execution, query, eventEmitter, resultAdder) {
```
Comment (pheyvaer) [inline src/right-field/RightField.jsx:260]: `resultAdder` is not in the docs.

File: README.md (hunk 1 above)
Comment (AronBuzogany) [reply README.md]: I've made a seperate object for this in [commit 3af2e39](https://github.com/SolidLabResearch/generic-data-viewer/pull/42/commits/3af2e3971f964d6f0d92a2690bd045c48e6cd079)

File: src/right-field/RightField.jsx (hunk 2 above)
Comment (AronBuzogany) [reply src/right-field/RightField.jsx:261]: This name was outdated in the documentation, I have updated it in [commit e57020e](https://github.com/SolidLabResearch/generic-data-viewer/pull/42/commits/e57020ec88726cd6438ed3c708f54a09b23710d4)

**Review by pheyvaer — APPROVED** [review]

**Pull request merged**